test/services/apimap-sender.test.js

```diff
@@ -0,0 +1,75 @@
+const ApplicationContext = require('../../src/context/application-context');
+const ApimapSender = require('../../src/services/apimap-sender');
+
+describe('services > apimap-sender', () => {
+  const context = new ApplicationContext();
+  const superagentRequestEndFunction = jest.fn();
+  context.init((ctx) => ctx
+    .addInstance('forestUrlGetter', jest.fn())
+    .addInstance('logger', { warn: jest.fn(), error: jest.fn() })
+    .addInstance('superagentRequest', {
+      post: () => ({ send: () => ({ set: () => ({ end: superagentRequestEndFunction }) }) }),
+    })
+    .addClass(ApimapSender));
+
+  const { apimapSender, logger } = context.inject();
+
+
+  describe('send', () => {
+    it('should post content then get response from forestUrlGetter', () => {
```

Comment:
[Suggestion] test label should be updated here, since `forestUrlGetter` does not exist anymore :)